src/lib/y2network/sysconfig/connection_config_readers/eth.rb

```diff
@@ -0,0 +1,49 @@
+# Copyright (c) [2019] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "y2network/connection_config/ethernet"
+
+module Y2Network
+  module Sysconfig
+    module ConnectionConfigReaders
+      # This class is able to build a ConnectionConfig::Ethernet object given a
+      # Sysconfig::InterfaceFile object.
+      class Eth
```

Comment:
`class Ethernet` for consistency with the ConnectionConfig namespace?

Reply:
we plan to address it in this sprint with unified InterfaceType everywhere which will have method for consistent names.

Reply:
addressed in https://github.com/yast/yast-network/pull/888 I think we should postpone this change as it is quite big and can invalidate testing we did.